### DISCUSSION TITLE
C API example

### DISCUSSION THREAD

**Discussion by sDos280** [electronstudio/raylib-python-cffi#85] [Ideas]
I have an idea, I want to make an example of a basic window example in pyray using the C API. 
I know that pyray uses the python API for most of his examples, but sometimes that isn't the case like in the shader examples, so I was thinking about making an example on the C API so it will be (hopefully) clear on how to use that.

please give me your opinion.

**Comment by electronstudio** [comment]
Yes go ahead. Most people use the pyray api so most of the examples use it but I will accept either or both.

**Comment by sDos280** [comment]
is that example ok?

https://github.com/sDos280/pyray-C-API-example/blob/main/Capi_basic_window.py

if it does, how should I call the file, and where to put it?
